skyportal/tests/fixtures.py

```diff
@@ -104,16 +104,19 @@ def groups(obj, create, passed_groups, *args, **kwargs):
         instruments = [InstrumentFactory(), InstrumentFactory()]
         filters = ['ztfg', 'ztfr', 'ztfi']
         for instrument, filter in islice(zip(cycle(instruments), cycle(filters)), 10):
+            np.random.seed()
             phot1 = PhotometryFactory(obj_id=obj.id,
                                       instrument=instrument,
                                       filter=filter,
-                                      groups=passed_groups)
+                                      groups=passed_groups,
+                                      alert_id=np.random.randint(100, 9223372036854775807))
             DBSession().add(phot1)
             DBSession().add(PhotometryFactory(obj_id=obj.id, flux=99.,
                                               fluxerr=99.,
                                               instrument=instrument,
                                               filter=filter,
-                                              groups=passed_groups))
+                                              groups=passed_groups,
+                                              alert_id=np.random.randint(100, 9223372036854775807)))
```

Comment:
See comment above about negative ints

Reply:
I think it's fine to only have positive numbers for the test fixture instances

Reply:
> I propose that alert_id defaults to `None`, and that the user-facing api have a flag added called "assign alert_id" which assigns an alert_id if requested and alert_id is unspecified in the post packet.

@dannygoldstein the user no longer specifies the alert_id, it is computed via `hash((obj_id, mjd))`